service/resource/namespace/current.go

```diff
@@ -17,6 +17,12 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
+	// No-op if we are not using cloudformation.
+	if !key.UseCloudFormation(customObject) {
+		r.logger.LogCtx(ctx, "debug", "not processing Kubernetes namespace")
+		return nil, nil
+	}
+
```

Comment:
Again adds a no-op for getting current state if we're not using Cloud Formation.